config/config.js

```diff
@@ -0,0 +1,23 @@
+export default {
+  development: {
```

Comment:
How do I verify that this works?

Reply:
@adesege  You'll need to create a model in order to test if it works, do like this
```npx sequelize-cli model:generate --name workers --attributes firstName:string,lastName:string,email:string```
it's just one command

Reply:
This won't work locally because dotenv is not initialized locally. Have you tested this locally and you are sure it works?